src/components/10-atoms/input-text/README.md

```diff
@@ -73,10 +74,27 @@ The string-valued `label` provides the label text as HTML.
 
 The string-valued `refId` sets the reference for label and input, if no `refId` is set a random id number will be created.
 
-### name\*
+### name (required within `<form>` tag)
 
 The string-valued `name` of the element for purposes of form submission.
 
+### maxLength
+
+Defines a maximum number of characters, that a user has available. Attention: If the user uses ALL available characters, a validation message will occur and the form should not be submittable. Therefore, the counter of available characters left displays always the value of `maxLength -1`. The reason for this strange behavior is, that another implementation is not possible without getting rid of the internal html `<textfield>` element, which would degrade the accessibility rating of the component.
+
+### counter
+
+Defines the text that the user will see, when he types into an input field that has the `maxLength`-attrbute set.
+This can have several options:
+
+- Leave it empty, so only the numbers of characters left will be displayed.
+- Set it to "characters left" to get a message formatted like this: `9 characters left`.
+- Set the counter between the text: "Still ##counter## characters left" will result into `Still 9 characters left'`. Keep in mind that the user can also see the number "0", so adapt the text so this will not look weird. `Still 0 characters left` does not sound so good.
```

Comment:
I think the explanation must go beyond examples to describe the general case, cf. axa-textarea README.